src/main/java/uk/gov/pay/ledger/event/model/EventDigest.java

```diff
@@ -63,6 +68,7 @@ public static EventDigest fromEventList(List<Event> events) {
                 latestEvent.getResourceType(),
                 latestEvent.getResourceExternalId(),
                 latestEvent.getParentResourceExternalId(),
+                TransactionState.fromEventType(latestSalientEventType).getState(),
```

Comment:
I thought it might be nice if the event digest reports on the event side of things (i.e. latest event) and the transaction side of things does the conversion to `TransactionState` - this is how it was before. Thoughts?

Reply:
Yeah that separation makes sense, will update